src/testdir/test_terminal3.vim

```diff
@@ -793,7 +791,8 @@ func Test_terminal_sync_shell_dir()
   let tmpfolder_url = fnamemodify(tempname(),':h').'/'.chars_url
   call mkdir(tmpfolder, "p")
   let buf = Run_shell_in_terminal({})
-  call term_sendkeys(buf, "echo -ne $'\\e\]7;file://".tmpfolder_url."\\a'\<CR>")
+  call TermWait(buf)
```

Comment:
I think the call to TermWait() should be moved into the Run_shell_in_terminal() function itself so that
this helps in other places where this function is called.